data/coalition/coalition missions.txt

```diff
@@ -143,3 +143,842 @@ mission "Coalition License Patch"
+			choice
+				`	"I can help with that. Do you have any type of ship in mind?"`
+				`	"Sorry, but I won't be heading back to human space for a while."`
+					decline
+			`	He tells you about how this comic series is set during a less developed time, so he has been trying to take after older ship designs.`
+			`	After you briefly detail the human shipyards to him, he is most interested in Betelgeuse's older, historical designs, and settles for those.`
+			`	He thanks you and wishes you safe travels on your journey to <planet>.`
+				accept
+
+
+
+mission "Arachi Cartoonist 2"
+	name "A New Design"
+	description "Return to Drugub on <destination>, and give him the flyer with pictures and some details of Betelgeuse's ship models."
+	landing
+	to offer
+		has "Arachi Cartoonist 1: done"
+	source "Prime"
+	destination "Mebla's Portion"
+	on offer
+		conversation
+			`When you ask the shipyard clerk if you could have something that detailed their ship models to "show a friend who is new to spaceships", she hands you a flyer, which you then unfold to see all of Betelgeuse's ships depicted, as well as some details on each (and the pricing for each too, of course).`
+			`	"Usually we show these to first time buyers, but you can take one if you want. Are you here for a friend like that? Maybe I could help you figure out what ship would best suit them."`
+			`	You thank her, saying that your friend does not have the funds for any of the ships, and just wants some info, and head back to your ship.`
+			`	Hopefully a small pamphlet will do for Drugub to get some ideas.`
+				accept
+	on complete
+		"coalition jobs" ++
+		payment 75940
+		conversation
+			`Drugub is excitedly waiting for you when you return. However simple a piece of paper it may seem to you, he grabs the flyer as if it's some scroll of knowledge.`
+			`	He hands you your payment of <payment>, and spends some time with you still, asking you all the questions he can think of about each ship.`
+			`	"Very beautiful, this 'Leviathan' is. Thinking of many ways to draw it, I already am," he says. "Maybe a flagship of one of my characters, make it I will..."`
+			`	He spends some more minutes listening to you after he asks that you describe how you've seen each of the ships perform, thanks you again, and prepares to leave.`
+			`	"Oh, before I go, Captain <last>, use the name of your own ship for that 'Leviathan', may I? Just as a small thank you."`
+			choice
+				`	"Of course! It's called <ship>."`
+				`	"The name is nothing special, I wouldn't even get the reference of it if I read your comic in the future honestly."`
+					goto noship
+			`	"<ship> it is! Do it justice I will, Captain. Thanks again."`
+				accept
+			label noship
+			`	"Ah, I see. Well, just a reference it would be, so as you say not very special it would be I suppose. Anyway, thanks again Captain."`
```

Comment:
```suggestion
			`	"Ah, I see. Well, just a reference it would be, so as you say, not very special it would be, I suppose. Anyway, thanks again, captain."`
```
Don't really like that middle sentence. You might consider rewriting it in a more concise way.

Reply:
changed the middle a bit; same thing on Captain as previously mentioned